resources/party.ts

```diff
@@ -15,6 +17,184 @@ const emptyRoleToPartyNames = () => {
   };
 };
 
+// TODO: de fr ko
+const roleLocalized: Record<Role, Partial<Record<Lang, string>>> = {
+  tank: { ja: 'タンク', cn: '坦克' },
+  healer: { ja: 'ヒーラー', cn: '治疗' },
+  dps: { ja: 'DPS', cn: '输出' },
+  crafter: { ja: 'クラフター', cn: '能工巧匠' },
+  gatherer: { ja: 'ギャザラー', cn: '大地使者' },
+  none: { ja: 'すっぴん士', cn: '冒险者' },
+};
+
+// TODO: ja(abbr), de, fr, ko
+const jobLocalized: Record<Job, Record<'abbr' | 'full', Partial<Record<Lang, string>>>> = {
+  NONE: {
+    abbr: { ja: 'すっぴん士', cn: '冒险' },
+    full: { en: 'Adventurer', ja: 'すっぴん士', cn: '冒险者' },
+  },
+  GLA: {
+    abbr: { ja: '剣術士', cn: '剑术' },
```

Comment:
Are you lacking of English abbreviations here? And Japanese has their abbreviations for most jobs in just one or two characters rather than call out the full names, e.g. 剣 for 剣術士, ナ for ナイト or 暗 for 暗黒騎士. If you're not sure about that, maybe you could leave it blank and Japanese folks would fill them up once the PR got merged.